### DISCUSSION TITLE
Automatically sync 'watched' status between 2 Sage servers?

### DISCUSSION THREAD

**Discussion by JustFred51** [OpenSageTV/Community#39] [Windows]
I'm running Sage servers (v9) on 2 separate Windows boxes.  I'd like to automatically sync the recordings' Watched status between them.  Syncing once a day (probably at night) is good enough.   Anybody doing this?  Google searches haven't provided ideas. 
**--------------------------------------**
> **System #1:** Win7-64, I7-4970, 16 GB mem, 8TB HD.  Java-64 1.8.0_141.  Sage-64 v9.2.1  ATSC: 2x HDHR-US (1st gen white) tuners.  HD-200. 
> **System #2:** Win7-64, I7-920, 8 GB mem, 8TB HD.  Java 1.8.0_131.  Sage v9.1.6.747.  ClearQAM: 2x HDHR3-US tuners.  HD-200.
> **System #3:** Win7-64, I7-920, 12 GB mem, 4TB HD.  Java-64 1.8.0_141.  Sage-64 v9.2.1  ATSC: 2x HVR2250; Spectrum Cable + network encoders & USB-UIRT.  3x HD-200.
>

**Comment by RJ-Make** [comment]
Not sure if this is what you're looking for but.... I run this as a part of my post recording media processing batch file (Mastered by MCEBuddy)

With default Password Use:
```
"%~dp0wget.exe" --delete-after --tries=3 --directory-prefix="c:\temp" "http://sageTV:frey@127.0.0.1:7070/sage/AiringCommand?command=SetWatched&MediaFileId=%mediafileid%"
"%~dp0wget.exe" --delete-after --tries=3 --directory-prefix="C:\temp" "http://sageTV:frey@127.0.0.1:7070/sage/GlobalCommand?command=UpdateMediaLibrary"
```

Without Password Use
```
"%~dp0wget.exe" --delete-after --tries=3 --directory-prefix="c:\temp" "http://127.0.0.1:7070/sage/AiringCommand?command=SetWatched&MediaFileId=%mediafileid%"
"%~dp0wget.exe" --delete-after --tries=3 --directory-prefix="C:\temp" "http://127.0.0.1:7070/sage/GlobalCommand?command=UpdateMediaLibrary"
```

**Reply by JustFred51** [reply]
Web access isn't built-in to Sage.  Does this require installation of the Jetty plugin?   I haven't been able to find the Jetty documentation;  all the download links in the old Sage forums are dead.
**--------------------------------------**
> **System #1:** Win7-64, I7-4970, 16 GB mem, 8TB HD.  Java-64 1.8.0_141.  Sage-64 v9.2.1  ATSC: 2x HDHR-US (1st gen white) tuners.  HD-200. 
> **System #2:** Win7-64, I7-920, 8 GB mem, 8TB HD.  Java 1.8.0_131.  Sage v9.1.6.747.  ClearQAM: 2x HDHR3-US tuners.  HD-200.
> **System #3:** Win7-64, I7-920, 12 GB mem, 4TB HD.  Java-64 1.8.0_141.  Sage-64 v9.2.1  ATSC: 2x HVR2250; Spectrum Cable + network encoders & USB-UIRT.  3x HD-200.

**Comment by TGsell1** [comment]
Yes you need the Jetty plug-in and the web UI is pretty good although I can’t playback from it but it’s good for remote access for scheduling and other functions.  Sent from my iPhoneOn Jun 16, 2023, at 7:42 PM, Keith Fischer ***@***.***> wrote:﻿
Web access isn't built-in to Sage.  Does this require installation of the Jetty plugin?   I haven't been able to find the Jetty documentation;  all the download links in the old Sage forums are dead.
--------------------------------------

System #1: Win7-64, I7-4970, 16 GB mem, 8TB HD.  Java-64 1.8.0_141.  Sage-64 v9.2.1  ATSC: 2x HDHR-US (1st gen white) tuners.  HD-200.
System #2: Win7-64, I7-920, 8 GB mem, 8TB HD.  Java 1.8.0_131.  Sage v9.1.6.747.  ClearQAM: 2x HDHR3-US tuners.  HD-200.
System #3: Win7-64, I7-920, 12 GB mem, 4TB HD.  Java-64 1.8.0_141.  Sage-64 v9.2.1  ATSC: 2x HVR2250; Spectrum Cable + network encoders & USB-UIRT.  3x HD-200.
